src/RoslynInsertionTool/RoslynInsertionTool/CoreXT.cs

```diff
@@ -41,21 +51,57 @@ public static async Task<CoreXT> Load(GitHttpClient gitClient, string commitId)
                 DefaultConfigPath,
                 download: true,
                 versionDescriptor: vsBranch);
-
             var defaultConfigOriginal = await new StreamReader(defaultConfigStream).ReadToEndAsync();
 
+            string? legacyPropsOriginal;
+            try
+            {
+                using var legacyPropsStream = await gitClient.GetItemContentAsync(
+                    vsRepoId,
+                    LegacyProjectPropsPath,
+                    download: true,
+                    versionDescriptor: vsBranch);
+
+                legacyPropsOriginal = await new StreamReader(legacyPropsStream).ReadToEndAsync();
+            }
+            catch (VssServiceException)
```

Comment:
is there a particular property on the exception / exception subtype we could look for that means the file did not exist (versus some other random error

Reply:
or at least logging the exception I think would be useful, in case we see missing changes to that legacy props file that shouldn't be missing

Reply:
Unfortunately, I couldn't find a way to more specifically identify a "not found", versus a network failure, for example.

Reply:
I added some log statements in d518717. The output looks like this

```
New Insertion Into rel/d16.8 Started

Verifying given authentication for https://devdiv.visualstudio.com/DefaultCollection
Creating VisualStudioRepoConnection object from https://devdiv.visualstudio.com/DefaultCollection
Verification succeeded for https://devdiv.visualstudio.com/DefaultCollection
Creating ComponentBuildConnection object from https://dev.azure.com/dnceng/
Verifying given authentication for https://dev.azure.com/dnceng/
Verification succeeded for https://dev.azure.com/dnceng/
Get Latest Passed Component Build
Getting latest passing build for project internal, queue dotnet-roslyn CI, and branch main-vs-deps
Found build number 20210630.1
Downloading artifacts to C:\Users\rikki\AppData\Local\Temp\Roslynmain-vs-deps\VSSetup
Artifact download took 20 seconds
Unable to load LegacyProjects.props. It will not be updated in this insertion.
TF401174: The item 'src/ConfigData/Packages/LegacyProjects.props' could not be found in the repository 'VS' at the version specified by '<Commit: 3a4f23a782c88da2528f578230ad991f365a2d56 >' (resolved to commit '3a4f23a782c88da2528f578230ad991f365a2d56')
...
```